Makefile

```
@@ -57,6 +62,32 @@ install-go-test-coverage:
+clean-e2e-images: ## Clean containers for previous e2e runs
+	$(CONTAINER_TOOL) images | awk '{if (length($1)==54)print $1":"$2}' | xargs $(CONTAINER_TOOL) rmi
+	$(CONTAINER_TOOL) images | grep '^<none>' | awk '{print $3}' | xargs $(CONTAINER_TOOL) rmi
+
+.PHONY: fast-e2e-test
+fast-e2e-test:
 	cd ./tests/etcdv3 && go test -count=1 ./...
+
+.PHONY: full-e2e-test
+full-e2e-test: build-images clean-e2e-containers clean-images fast-e2e-test
+
+.PHONY: e2e-test
```

checkmake (0.2.2)

[warning] Missing required phony target "clean"

(minphony)